### PR TITLE
Followers and Following added

### PR DESCRIPTION
**Pull request by kylecornelissen**
- Created models for github_follower and github_following
- Created partials for github_followers and github_following
  -- Add partials to users show page  

- Added render_followers and render_following to github_facade

### CODE REVIEW

File: spec/features/user/user_can_see_github_repos_spec.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@
       within('#github') do
         page.assert_selector('.repo', count: 5)
         within(first('.repo')) do
-          expect(page).to have_content('test_user/repo1')
+          expect(page).to have_content('alexander-mathieu/little_shop_v2')
```
Comment (Myrdden) [inline spec/features/user/user_can_see_github_repos_spec.rb:21]: This will fail if anyone except for you is running the test that's why we mock it out.

**Review by Myrdden — COMMENTED** [review]
Potential refactor- Merge GithubFollower and GithubFollowing into something like GitHubUser because they're doing the same thing.

File: spec/features/user/user_can_see_github_repos_spec.rb (hunk 1 above)
Comment (kylecornelissen) [reply spec/features/user/user_can_see_github_repos_spec.rb:21]: The cassette will keep all that info in there for the next user. I will create a mock token though.

**Review by Myrdden — APPROVED** [review]
gr8

**Pull request merged**